content/en/contribute/code/_index.md

```diff
@@ -22,15 +22,15 @@ The CHT community welcomes first-time contributors and experts alike. All commen
 
 ### First time contributor?
 
-Issues labeled [help wanted](https://github.com/medic/cht-core/labels/Help%20wanted) are a great place to start. Looking for other ways to help? You can also:
+Issues labeled [help wanted](https://github.com/medic/cht-core/labels/Help%20wanted) are a great place to start. 
```

Comment:
I think this should be part of the list below, rather than a starting point. Suggestion below...

Reply:
```suggestion
```

Reply:
Ya, makes sense. I think it would then need a bigger edit than the [other suggestion](https://github.com/medic/cht-docs/pull/876#discussion_r1033993128) to be consistent in tone, so will leave it for another pass.